### PR TITLE
Project 5, Guess Who

### PR DESCRIPTION
**Pull request by MarySparrow**
Maria Sparre 
live site: 
https://project5guesswho.netlify.app/

### CODE REVIEW

**Review by honkoan — APPROVED** [review]
I like your Guess Who game! You did great job with this project. If you struggled a lot with this project you also learned a lot and I can see it when I'm looking your code and your game. So the struggling was worth of it! Your code is clean and easy to understand and the game works as it should work. The flip card effect is cool! Be proud of yourself! 🙌🏻

File: code/script.js [hunk 1]
```diff
@@ -247,97 +250,129 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(secret)
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:253]: Yes, all console.log()s, commented-out lines of code and tips from teachers hidden in comments should be removed from the final version 🙈

File: code/style.css [hunk 2]
```diff
@@ -92,6 +93,7 @@ select {
 
 .card:hover {
   background-color: var(--secondary);
+  transform: rotateY(180deg)
 }
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/style.css:98]: 😻

**Review by KarolCislowski — APPROVED** [review]
You created an impressive project 🌈 This flip animation was a very nice surprise 🙊  

Remember that you always can revisit this project to do more stretch goals and make it even better 🦾

**Pull request closed without merging**